### ISSUE TITLE
Training PP-OCRv3 on one-word images

### ISSUE THREAD

**Issue by bely66** [PaddlePaddle/PaddleOCR#8152]
Hi Everyone,
I'm training the ppocrv3 recognition for 9M images dataset with various fonts and augmentations.

What I'm looking for is faster training and higher accuracy
And my dataset images only consists of one word images instead of sentences.

So what I'm thinking is 
1. Reducing the input image size to (32, 160) instead of (48, 320)
2. Removing the RecConAug as it adds longer or reduce the max word char to about 16

My accuracy currently is about 83% after 16 epochs and it has taken almost 5 days to complete on 2xV100 compute.
The dataset is pretty straight forward and easily get above 95% accuracy on models like vanilla CRNN and SAR

So would this strategy affect accuracy negatively? and is there any other way to optimize training time because it's very costly to train 200 epochs for 50 days?

**Comment by bely66** [comment]
@an1018 
Hi thanks for your reply
i won’t be able to finetune because the language and dictionary I’m using is custom and new 

my issues are training from scratch for a long time

**Comment by tink2123** [comment]
Yes, if your training data is very simple, the v3 strategy may have a negative impact, you can try to remove the GTC strategy, refer to https://github.com/PaddlePaddle/PaddleOCR/issues/6355#issuecomment-1134356635 , and remove conaug at the same time, if you do not need additional data enhancement.

**Comment by rm-asif-amin** [comment]
@bely66 Did you get a solution to this? I'm also Trying to train for a new language with new dict(I added characters to an English dict). Ultimate goal is to have a bi-lingual model. I've trained on 75k one-word images for 2 days and this is what my training and eval accuracy looks like-

![image](https://github.com/PaddlePaddle/PaddleOCR/assets/50940878/8a1c6098-f550-44ed-97c7-b0866c9a05cc)

My questions are-
1. Is the model actually learning? It seems like it is.
2. Is this a training length issue? Or do i not have enough data? Managing more data will be very tough. 
3. What would you suggest for 95% + accuracy?

<details>
  <summary>My current architecture-</summary>


Architecture:
  model_type: rec
  algorithm: CRNN
  Transform: null
  Backbone:
    name: MobileNetV3
    scale: 0.5
    model_name: small
    small_stride:
    - 1
    - 2
    - 2
    - 2
  Neck:
    name: SequenceEncoder
    encoder_type: rnn
    hidden_size: 48
  Head:
    name: CTCHead
    fc_decay: 1.0e-05
Loss:
  name: CTCLoss
PostProcess:
  name: CTCLabelDecode
Metric:
  name: RecMetric
  main_indicator: acc
  
  </details>
